libraryBuilds/joda-primitives/joda-convert/build.gradle

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2015 the authors of j2objc-common-libs-e2e-test
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencies {
+    j2objcTranslation 'org.joda:joda-primitives:1.0:sources'
```

Comment:
Add comment explaining that joda-primitives depends on joda-convert. 

Putting this comment on PR #28 to see if this is preserved while other comments aren't.